modules/lib/src/main/resources/assets/js/app/settings/wizard/panel/form/element/ProjectAccessControlEntryView.ts

```diff
@@ -51,10 +63,23 @@ export class ProjectAccessControlEntryView
                 this.item.setAccess(event.getNewValue());
                 this.notifyValueChanged(this.getItem());
             });
+            this.accessSelector.onClicked(() => this.closeOtherAccessSelectors());
             this.appendChild(this.accessSelector);
         }
         this.accessSelector.setValue(this.item.getAccess(), true);
 
         this.appendRemoveButton();
     }
+
+    private closeOtherAccessSelectors(): void {
```

Comment:
I think the better solution would be to check `TabMenu.hideMenuOnOutsideClick` method